api-docs/rst/dev-guide/release-notes.rst

```diff
@@ -0,0 +1,19 @@
+.. _release-notes:
+
+=============
+Release Notes
+=============
+Learn about new features, enhancements, known issues, resolved issues, and
+other important details about |apiservice| |version| service updates.
+
+.. note:: For information about using the API, see the
+   :ref:`documentation overview <index>`.
+
+.. toctree::
+   :maxdepth: 2
+
+   November 20, 2013 <release-notes/as_20131120.rst>
```

Comment:
In the index.rst files, you don't have to include the *.rst extension in the file list.  If the build doesn't give any errors, then I think it isn't a problem, but can you verify the build.